Applications/ConsumerApi/test/ConsumerApi.Tests.Integration/Features/Relationships/{id}/Revoke/PUT.feature

```diff
@@ -12,8 +12,8 @@ User revokes a Relationship
 
     Scenario: Revoke Relationship to an Identity in status "ToBeDeleted"
         Given Identities i1 and i2
-        And a pending Relationship r between i1 and i2 created by i2
+        And a pending Relationship r between i1 and i2 created by i1
```

Comment:
I agree that `i1` is correct here.
But did you check out why it even worked before your fix? I assume that the step definition's implementation contains a bug.

Reply:
It worked because `EnsureRelationshipRequestIsCreatedBySelf(activeIdentity);` was giving back 400. The code didn't even get to part where checks the status. This is how I figured out that I have to change from `i2` to `i1`.

Reply:
I switched to `main` branch and just changed `i2` to `i1` and the test passed also.

Reply:
> I switched to `main` branch and just changed `i2` to `i1` and the test passed also.

Okay, but this confirms my assumption about the incorrect implementation of the step definition.